### PR TITLE
[pytorch] use randn in dropout

### PR DESCRIPTION
**Pull request by mli**
simply the code, we should avoid use _ functions

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Review by AnirudhDagar — APPROVED** [review]
LGTM! Probably failed in the `tensorflow` tab.

**Comment by astonzhang** [comment]
Just triggered re-running (due to randomness in tf)

**Comment by mli** [comment]
Job d2l-en/PR-1714/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1714/
